src/components/calendar-components/AddEvent.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {ParticipantsSelect} from './ParticipantsSelect';
+import {TypeSelect} from './TypeSelect';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import {getDateFormatYMD} from './../../helpers/calendar-helper';
+
+
+export const AddEvent = (props) => {
+  const date = props.date;
+  return (
+    <div className="side-display">
+      <h1>Legg til ny hendelse</h1>
+      <form>
```

Comment:
Conventionally, fields that belong to the same form should be contained in one `<form>`. What happens if we do that?